[ComicsKingdomBridge] Fixes to accomodate new layout and site changes

TReKiE opened the pull request:
ComicsKingdomBridge started failing a few days ago with "Call to a member function getAttribute() on null", as the site layout has changed significantly, making the prior method of obtaining the previous days comics obsolete.  The ComicsKingdom new site also has a new name, which is also changed in this PR to avoid unnecessary redirects.

Note: This PR does not address that the new site's SSL certificate is not fully configured resulting in, "cUrl error: SSL certificate problem: unable to get local issuer certificate" on some systems.  That can be solved by installing the "GoDaddy Secure Server Certificate (Intermediate Certificate) - G2" into your local machine's CA store.

TReKiE committed:
Change where the link is found in the page to the new location

Additional code to verify there's a link on the page, the site seems to only go back three days now, however the og:image property still exists on older days, so it appears to be a bug on their end.

TReKiE committed:
Fix for URI not matching the RSS item/comic

The link for the next comic was being found before specifying the URI for the RSS item, now fixed by moving the code down to after the item is created.

TReKiE committed:
Update URL slightly by removing www

The new site redirects to the non-www on each request, performance improvement by updating the site's URI.

em92 commented:
Hi, @TReKiE! Could you please add exampleValue to 'comicname' parameter? I don't know what to input there to test it.

TReKiE committed:
Add exampleValue and title instructions for comicname

TReKiE commented:
Sure thing, added a personal favourite as an example, and some guidance in the title on what to put in.

em92 commented on bridges/ComicsKingdomBridge.php:
```diff
@@ -41,6 +41,8 @@ public function collectData(){
 			$item['content'] = '<img src="' . $imagelink . '" />';
 
 			$this->items[] = $item;
+			$link = $page->find('div.comic-viewer-inline a', 0)->href;
```
Notice: Trying to get property 'href' of non-object in /var/www/rss-bridge/pr2444/bridges/ComicsKingdomBridge.php on line 42
https://feed.eugenemolotov.ru/pr2444/?action=display&bridge=ComicsKingdom&comicname=mutts&format=Html

TReKiE replied:
My apologies, I had display_errors off and failed to read the error log.  As one might expect, this notice was occurring because the site is currently not working properly when going back more than 3 days of comics, as doing so results in a blank page.  I was hoping it might fix itself by now as more comics arrived, but that hasn't happened.

I have therefore changed the loop in 9fda15e to be only 3 days, but still left in the error handling, as it will still fatally fail if the site changes again, but only write a notice in the unlikely event it can't get all 3 days worth of comics.

em92 replied:
Nevermind. Instead of setting up display_errors, you can just create empty DEBUG file in rss-bridge root directory.

TReKiE committed:
Change to only grab the last 3 comics (from 5)

Going back further than 3 days generates a blank page, causing a notice.  Soft error handling still in place as a fatal error doesn't seem necessary when dealing with previous days comics.

em92 commented:
gj, @TReKiE !
